.gitignore

```diff
@@ -22,3 +22,4 @@
 .yardoc/
 /doc/
 /failures
+.DS_Store
```

Comment:
I'd prefer to make repo's easier to contribute to for new contributors since this is just boilerplate that can be part of all repos (like most open source projects do).

All dependencies should be self contained with locally with each repo, for example we wouldn't expect people to run their own linter.

That said I imagine this is one of those 'tabs vs spaces' things so will change if needed.

tl;dr contributors > conventions